docs/0.1/creating_products.md

```diff
@@ -0,0 +1,315 @@
+
+Finally, this procedure shows how to [update a product](#update-a-product)
+(with `grid product update`) and [delete a product](#delete-a-product) (with
+`grid product delete`) when it is no longer needed.
+
+## Prerequisites
+
+* A Splinter network with at least two nodes.
+
+* An approved Splinter circuit that has two (or more) member nodes.
+
+* The full service ID (also called the "full circuit ID") for this circuit, in
+  the format `*circuitID*::*serviceID*`.
+  This procedure uses the example ID `01234-ABCDE::xyZZ`.
+
+  Tip: Use `splinter circuit list` to display the circuit and service IDs.
+  In the example Docker environment, run this command in the node's
+  `splinterd` container.
+
+* The Grid daemon's endpoint (URL and port) on one or both nodes.
+  The example environment uses `https://localhost:8080`.
+
+* An organization ID and associated agent's public key for an existing Grid
+  organization. This procedure assumes that you are the agent (your public key
+  is used to identify the agent).
+
+  Tip: You can use `curl` to request organization and agent information from
+  the Grid REST API, as in these examples:
+
+  `$ curl https://localhost:8080/organization?service_id=01234-ABCDE::xyZZ`
+
+  `$ curl http://localhost:8080/agent?service_id=01234-ABCDE::xyZZ`
+
+See [Running Hyperledger Grid on Splinter]({% link docs/0.1/grid_on_splinter.md
+%}) for the procedures to [set up and run
+Grid](/docs/0.1/grid_on_splinter.html#set-up-and-run-grid) in a set of Docker
+containers, [create a
+circuit](/docs/0.1/grid_on_splinter.html#create-a-circuit), and [create an
+organization](/docs/0.1/grid_on_splinter.html#demonstrate-grid-smart-contract-functionality)
+in the example Grid-on-Splinter environment.
+
+## Procedure
+
+**IMPORTANT**: The commands in this procedure show host names, IDs, Docker
+container names, and other values from the example Grid-on-Splinter environment
+that is defined by
+[`grid/examples/splinter/docker-compose.yaml`](https://github.com/hyperledger/grid/blob/master/examples/splinter/docker-compose.yaml).
+This file sets up the nodes `alpha-node-000` and `beta-node-000` and runs the
+Grid and Splinter components in separate containers; these names appear in
+example prompts and commands. (See [Running Hyperledger Grid on
+Splinter]({% link docs/0.1/grid_on_splinter.md %}) for more information.)
+
+If you are not using this example environment, replace these items with the
+actual values for your environment when entering each command.
+
+### Connect to a Splinter node
+
+1. Connect to one of the Splinter nodes and start a bash session.
+
+   For the example environment, use this command to connect to the `gridd-alpha`
+   container and run Grid commands on `alpha-node-000`.
+
+   ```
+   $ docker exec -it gridd-alpha bash
+   root@gridd-alpha:/#
+   ```
+
+### Set Up Your Environment
+
+{:start="2"}
+
+2. If necessary, generate your public/private key files in `$HOME/.grid/keys/`.
```

Comment:
This feels weird to include here since before this key could be used, an agent with this key would need to be created for the organization in Pike. (assuming that the this key is not the one used to create the organization, which would automatically create an agent).

Reply:
I moved the key info to a prerequisite, to clarify that the same keys must be used to create the organization (which happens before this procedure starts).